public/src/client/account/edit.js

```diff
@@ -242,17 +247,20 @@ define('forum/account/edit', ['forum/account/header', 'translator', 'components'
 						if (!url) {
 							return;
 						}
-						
+
 						uploadModal.modal('hide');
-						
+
 						pictureCropper.handleImageCrop({
 							url: url,
 							socketMethod: 'user.uploadCroppedPicture',
 							aspectRatio: '1 / 1',
+							allowSkippingCrop: false,
+							restrictImageDimension: true,
+							imageDimension: ajaxify.data.profileImageDimension,
```

Comment:
It's handled on the API side. Returns 128 as default if not set in ACP. Part of this PR. 

https://github.com/pichalite/NodeBB/blob/fd0f0beaf22b300ca410551e0469a84b8af094e9/src/controllers/accounts/edit.js#L30